accessiblity.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: Accessibility
+
+permalink: /accessibility/
+---
+
+Flutter is committed to enable developers to make their Flutter apps accessible:
+usable for all users, including users with disabilities such as blindness or
+motor impairment.
+
+* Placeholder for TOC
+{:toc}
+
+# Flutter's accessibility support
+
+Flutter supports three components of accessibility support:
+
+* **[Large fonts](#large-fonts)**: Render text widgets with user specified font
+  font sizes
+* **[Screen readers](#screen-reader)**: Communicate spoken feedback about UI
+  contents
+* **[High contrast](#high-contrast)**: Render widgets with colors that have
+  sufficient contrast
+
+## Large fonts {#large-fonts}
+
+Both Android and iOS contain system settings for configure the desired font
```

Comment:
for configure --> for configuring
or
for configure --> to configure

Reply:
Done